src/movies.js

```diff
@@ -1,16 +1,123 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    const directorsList = movies.map(function(name) {
+            return name.director
+        })
```

Comment:
You should name your variable to what they are. If you map through an array of movies, each item is a movie (not a name). It will help you and others reading your code more easily.